### PR TITLE
Package as .deb

### PR DESCRIPTION
**Pull request by jcapona**
Add workflows and necessary files to package as a `.deb` file.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Build-Depends:
+ debhelper-compat (= 12),
+Standards-Version: 4.5.1
+Homepage: https://novnc.com
+
+Package: novnc
+Architecture: all
+Depends:
+ ${misc:Depends},
+ procps,
```
Comment (angusjfw) [inline debian/control:14]: We previously also found that novnc uses `netstat` command which is from `net-tools` package. Like procps this is included in common distributions of debian but not all (eg wsl, docker)

Comment (angusjfw) [reply debian/control:14]: There are other things [here](https://salsa.debian.org/openstack-team/third-party/novnc/-/blob/debian/yoga/debian/control), maybe take another look at that

File: debian/copyright [hunk 2]
```diff
@@ -0,0 +1,62 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: novnc
+Source: https://github.com/pi-top/noVNC
+Upstream-Contact: deb-maintainers@pi-top.com
```
Comment (angusjfw) [inline debian/copyright:4]: maybe we should include some reference to the original source? like they did here https://salsa.debian.org/openstack-team/third-party/novnc/-/blob/debian/yoga/debian/copyright

**Review by angusjfw — COMMENTED** [review]
Broadly looks good but as I said in comments I think it's worth taking another look at the [existing](https://salsa.debian.org/openstack-team/third-party/novnc) deb packaging to check we've not missed anything. Eg I don't think we need to include the python library but are we sure about that?

**Review by angusjfw — COMMENTED** [review]
Broadly looks good but as I said in comments I think it's worth taking another look at the [existing](https://salsa.debian.org/openstack-team/third-party/novnc) deb packaging to check we've not missed anything. Eg I don't think we need to include the python library but are we sure about that?

File: debian/control (hunk 1 above)
Comment (jcapona) [reply debian/control:14]: `novnc_proxy` is more simple than `launch.sh` so no other dependencies are required (other than `procps`)... just added `python3-novnc` and `python3-numpy` just to be safe

File: debian/copyright (hunk 2 above)
Comment (jcapona) [reply debian/copyright:4]: makes sense, I added a link to our fork but it's better to point to the original source

**Review by angusjfw — APPROVED** [review]

**Pull request merged**